packages/keyring-api/tsconfig.json

```
@@ -4,6 +4,6 @@
     "baseUrl": "./"
   },
   "references": [{ "path": "../keyring-utils/tsconfig.build.json" }],
-  "include": ["./src"],
+  "include": ["./src", "../keyring-snap-sdk/src/methods.ts"],
```

Not sure we need this? 🤔
```suggestion
  "include": ["./src"],
```